public/scripts/spotifyArtist.js

```diff
@@ -0,0 +1,42 @@
+(function(module){
+  var artist = {};
+  artist.all=[];
+  var artistTemplate = Handlebars.compile($('#artist-template').html());
+  artist.getArtists = function(callback) {
+
+    $.ajax({
```

Comment:
`$.ajax()` is a low-level jQuery function, and it would be more appropriate at this point to use a `$.get()`.

``` javascript
$.get('api.spotify/url', function(data) {
  // code all the functional success code things
}).done(function() {
  // code all the things after done
})
```